lib/htmlbuilder.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+const fs = require('fs');
+const Handlebars = require('handlebars');
+const path = require('path');
+
+
+module.exports.buildHtml = (auditData) => {
+    var template = fs.readFileSync(path.join(__dirname, './template.html'), 'utf8');
+
+    var errors = [];
+
+      auditData.forEach(element => {
```

Comment:
This a great use case for the Array.map function.
```
var errors = auditData.map((element) => {
...
}
```
Doing this means you don't need to have the `errors.push`.